tests/test_go_expvar.py

```diff
@@ -1,152 +1,224 @@
-import unittest
+from collections import defaultdict
+import copy
+import os.path
 import time
-import os
-from tests.common import load_check
-import simplejson as json
 
+from nose.plugins.attrib import attr
+from tests.common import AgentCheckTest
+import simplejson as json
 
-class TestGoExpVar(unittest.TestCase):
 
-    def setUp(self):
-        self.agentConfig = {
-            'version': '0.1',
-            'api_key': 'toto'
+def _get_data_mock(url):
+    with open(url, 'r') as go_output:
+        return json.loads(go_output.read())
+
+
+@attr(requires='go_expvar')
```

Comment:
If only one of the test cases requires the integration setup, you should put the `attr` on those test cases. This allows people to run unit tests quickly by running `ci:run`.